### PR TITLE
[HttpFoundation] Add Laravel as a backer to the README

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

Thanks to them! :pray: 

/cc @taylorotwell

### CODE REVIEW

File: src/Symfony/Component/HttpFoundation/README.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ HttpFoundation Component
...
 
+Sponsor
+-------
+
+The HttpFoundation component for Symfony 5.4/6.0 is [backed][1] by [Laravel][2].
+
+Laravel is a PHP web development framework that is passionate about maximum developer
+happiness. Laravel is built using a variety of bespoke and Symfony based components.
+
+Help Symfony by [sponsoring][3] its development!
```
Comment (wouterj) [inline src/Symfony/Component/HttpFoundation/README.md:15]: Let's use inline reference, like we use in all other links in this doc? :)

**Review by wouterj — COMMENTED** [review]
Thanks Taylor! :pray:

File: src/Symfony/Component/HttpFoundation/README.md (hunk 1 above)
Comment (nicolas-grekas) [reply src/Symfony/Component/HttpFoundation/README.md:15]: I copy/pasted from a similar mention in the Routing readme for 5.3.

**Review by welcoMattic — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**